dll/win32/shell32/precomp.h

```diff
@@ -12,6 +12,7 @@
 
 #include <windef.h>
 #include <winbase.h>
+#include <wingdi.h>
```

Comment:
Why is the wingdi.h header needed now?

Reply:
```txt
In file included from ../dll/win32/shell32/precomp.h:19,
                 from ../dll/win32/shell32/CDropTargetHelper.cpp:23:
../sdk/include/psdk/commdlg.h:264:2: error: 'LPLOGFONTA' does not name a type; did you mean 'LPPOINTS'?
  LPLOGFONTA lpLogFont;
  ^~~~~~~~~~
  LPPOINTS
```

Reply:
> wingdi.h

I wonder why that error is happening.
`LPLOGFONTA` should already exist as NT5.2.
Should `commdlg.h` include `wingdi.h` somehow?

> CDropTargetHelper.cpp

Is that the only error / affected file?